src/amo/components/LandingPage/index.js

```diff
@@ -214,20 +233,23 @@ export class LandingPageBase extends React.Component {
           footerText={html.featuredFooterText}
           footerLink={html.featuredFooterLink}
           header={html.featuredHeader}
+          loading={loading}
```

Comment:
I think it's best to do in a future patch but technically these loading states shouldn't be shared; there are three API requests so we could load the data as it came in. I guess we'd want to A/B test that experience but my first thought is we should display data as we load it; if the featured data for some reason takes an extra `400ms` to load we should still display the other stuff, so arguably each request should have its own loading property.